### ISSUE TITLE
Hyperlinks are outdated in Codebase structure section of CONTRIBUTING.md file

### ISSUE THREAD

**Issue by akhildevelops** [neuronika/neuronika#110]
Shows 404 error while clicking on any hyperlink present in [Codebase structure of CONTRIBUTING.md](https://github.com/neuronika/neuronika/blob/main/CONTRIBUTING.md#codebase-structure). Ex: [data](https://github.com/neuronika/neuronika/tree/main/src/data)

Update the links pointing to workspace members.

**Comment by frjnn** [comment]
Yes. I plan to redo all the docs in the near future.Sent from my iPhoneOn 8 Nov 2022, at 15:37, Akhil ***@***.***> wrote:﻿
Shows 404 error while clicking on any hyperlink. Ex: data
Update the links pointing to workspace members.

**Comment by akhildevelops** [comment]
I'll work on this. 😊
